### PR TITLE
[DO NOT MERGE] Test Benchmark Bot

### PR DESCRIPTION
**Pull request by shawntabrizi**
Testing the benchmark bot

[skip ci]

### CODE REVIEW

File: frame/contracts/src/weights.rs [hunk 1]
```diff
@@ -206,11 +206,11 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Contracts CodeStorage (r:1 w:1)
 	// Storage: Contracts PristineCode (r:0 w:1)
 	fn instantiate_with_code(c: u32, s: u32, ) -> Weight {
-		(479_578_000 as Weight)
+		(680_527_000 as Weight)
```
Comment (athei) [inline frame/contracts/src/weights.rs:209]: This is some non trivial weight change here (it happens to all benchmarks). Any idea why this is?

Comment (shawntabrizi) [reply frame/contracts/src/weights.rs:209]: This sounds like the changes that @joao-paulo-parity made are not insignificant then.

**Comment by athei** [comment]
Contracts benchmark run really long. I always meant to work on this. Sorry for this.

**Comment by joao-paulo-parity** [comment]
/benchmark runtime custom --chain=dev --steps=50 --repeat=20 --pallet=pallet_contracts --extrinsic="instantiate_with_code" --execution=wasm --wasm-execution=compiled --heap-pages=4096 --output=./frame/contracts/src/weights.rs --template=./.maintain/frame-weight-template.hbs

**Comment by joao-paulo-parity** [comment]
See https://github.com/paritytech/bench-bot/pull/53

**Comment by ggwpez** [comment]
bot rebase

**Pull request closed without merging**